packages/lib-classifier/src/components/Classifier/components/SubjectViewer/helpers/getFixedNumber/getFixedNumber.js

```diff
@@ -1,3 +1,6 @@
 export default function getFixedNumber(number, digits) {
+  if (!digits) {
+    console.log('getFixedNumber: Missing digits variable')
+  }
```

Comment:
[minor][rambling] It's good that we're checking for input here, but if we don't have `digits` (i.e. undefined), won't this function still return `NaN`? Hmm, but that said, I don't know what's the correct "fallback" return value should be if `digits` is undefined... eh, NaN is fine.

Reply:
My thoughts too 😆 For now, I wanted to return a fallback so the subject viewer wouldn't crash.